lib/Horde/Rdo/Base.php

```diff
@@ -128,7 +128,12 @@ public function __get($field)
             } elseif (!empty($this->{$rel['foreignKey']})) {
                 $this->_fields[$field] = $m->findOne($this->{$rel['foreignKey']});
                 if (empty($this->_fields[$field])) {
-                    throw new Horde_Rdo_Exception('The referenced object with key ' . $this->{$rel['foreignKey']} . ' does not exist. Your data is inconsistent');
+                    throw new Horde_Rdo_Exception(sprintf(
+                        'The referenced object of %s instance with key %s = %s does not exist. Your data is inconsistent',
+                        get_class($this),
```

Comment:
I'd prefer using the ::class magic property over get_class but otherwise I like this.